.github/workflows/release.yml

```
@@ -0,0 +1,268 @@
+  contents: read
+
+env:
+  PHP_VERSION: '8.2'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  lint:
+    uses: ./.github/workflows/.lint.yml
+    with:
+      packages: ${{ format('["{0}"]', inputs.package) }}
+      php-versions: ${{ format('["{0}"]', env.PHP_VERSION) }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)